docs/sources/enterprise/white-labeling.md

```diff
@@ -26,6 +26,10 @@ You can change the following elements:
 - Side menu top logo
 - Footer and help menu links
 - Fav icon (shown in browser tab)
+  > Only available in Grafana Enterprise v7.0+
```

Comment:
The way this is formatted, it is unclear if the note applies to Fav icon or to the items below. I recommend putting the (Grafana 7.0+) after each of the three new bulets.